experimental/null_safety/README.md

```diff
@@ -0,0 +1,97 @@
+# Null safety Flutter example
+
+This folder contains a demonstration of the upcoming Dart feature, sound null
+safety. For additional information about this feature, see
+https://dart.dev/null-safety.
+
+The folder contains two sample Flutter apps:
+
+  * `null_unsafe_app`: This contains a small weather app, which has three null
+    errors in its rendering code as a result of not using null safety.
+    
+    Try running the app (`flutter run` in a terminal, or the Run action in
+    VSCode/Android Studio), and see if the UI works. The null errors are
+    sporadic, so try hot reloading a bunch of times and experience the app
+    sometimes working and sometimes failing.
+
+  * `null_safe_app`: This contains the same weather app, but this time using
+    null safety. There are two copies of the source code:
+
+    `lib/badapp.dart`: This is the same source code as in the unsafe app. Try
```

Comment:
If the main educational point is to compare the two approaches, WDYT about removing null_unsafe_app entirely, and focusing on null_safe_app and the difference between its two primary Dart files?

Reply:
I thought about it, but I'm not sure that motivates the problem. Some might not understand how they are prone to null errors today. `null_unsafe_app` is meant to show that in a very viceral way, with an app that works some of the time and fails the rest of it.

In `null_safe_app`, perhaps my naming `goodapp.dart` and `badapp.dart` is misleading. They both have null safety enabled, it's just that one doesn't have the errors fixed yet. Perhaps they should be named `app_with_null_errors.dart` and `app.dart`?

Reply:
We could also rename the top-level folders to `with_null_safety` (`null_safe_app` currently), and `without_null_safety` (`null_unsafe_app` currently)

Reply:
Let's discuss this offline at some point in the coming week. Right now, you've got a clear description of things in the README, and the actual code is tight enough that people sophisticated enough to be looking at NNBD right now will have no trouble understanding what's going on.